src/core/libraries/kernel/threads/pthread.h

```diff
@@ -0,0 +1,359 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <forward_list>
+#include <list>
+#include <mutex>
+#include <semaphore>
+#include <shared_mutex>
+
+#include "common/enum.h"
+#include "core/libraries/kernel/time.h"
+#include "core/thread.h"
+#include "core/tls.h"
+
+namespace Core::Loader {
+class SymbolsResolver;
+}
+
+namespace Libraries::Kernel {
+
+struct Pthread;
+
+enum class PthreadMutexFlags : u32 {
+    TypeMask = 0xff,
+    Defered = 0x200,
```

Comment:
I believe this is a typo, but 'defered' is used for at least one other variable name, so I'm not sure